### ISSUE TITLE
Static dependencies are not being served

### ISSUE THREAD

**Issue by tjkemper** [revaturelabs/scheduler#413]
## Error message

```
Mixed Content: The page at 'https://scheduler.staging.revaturelabs.com/' was loaded over HTTPS, 
but requested an insecure stylesheet 

'http://ajax.googleapis.com/ajax/libs/angular_material/1.1.0/angular-material.min.css'. 
This request has been blocked; the content must be served over HTTPS.
```

## Possible solution

http://stackoverflow.com/questions/18251128/why-am-i-suddenly-getting-a-blocked-loading-mixed-active-content-issue-in-fire
